### ISSUE TITLE
Cannot load image in iOS 10.2

### ISSUE THREAD

**Issue by ray-z** [zyra/ionic-image-loader#49]
Hello, thanks for the awesome plugin. But I have problem of using it in iOS 10.2.

I tested using your example project, but the images cannot be loaded. The spin is there forever.

Below is my 'Ionic Info' output:
```
Cordova CLI: 6.4.0
Ionic Framework Version: 2.3.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.3.7
ios-deploy version: 1.8.6
ios-sim version: 5.0.8
OS: OS X El Capitan
Node Version: v6.9.1
Xcode version: Xcode 8.2.1 Build version 8C1002
```
Could you please give some suggestions?

Thanks!

**Comment by Shane-Lester** [comment]
Same issues.
Using Ionic View to test -no joy. Constant spinning.
Worked in previous versions. Have updated ionic and ionic-image-loader only and not changed the rest of the code base. 
'Ionic Info' is
`global packages:

    @ionic/cli-utils : 1.1.2
    Cordova CLI      : 7.0.1
    Ionic CLI        : 3.1.2

local packages:

    @ionic/app-scripts              : 1.3.7
   @ionic/cli-plugin-cordova       : 1.1.2
    @ionic/cli-plugin-ionic-angular : 1.1.2
    Ionic Framework                 : ionic-angular 3.2.1

System:

    Node       : v6.9.1
    OS         : Windows 10
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed
`

It was great previously. 
I don't have a Mac so can't run with Safari Inspector but if you need more info I can run logs if needed (please guide me to what info would be helpful)

Shane

**Comment by jonseg** [comment]
Same issues here.

```
Cordova CLI: 6.5.0
Ionic Framework Version: 3.0.1
Ionic CLI Version: 2.2.3
Ionic App Lib Version: 2.2.1
Ionic App Scripts Version: 1.3.4
ios-deploy version: 1.9.1
ios-sim version: 5.0.13
OS: OS X El Capitan
Node Version: v4.2.6
Xcode version: Xcode 8.2.1 Build version 8C1002
```

Are there any solution?
Thank you.

**Comment by ihadeed** [comment]
Hey guys,

Are you using WKWebView? if so, are you using the Ionic fork or the original one?

Try using the config method `setImageReturnType` to set the return type to `base64`. This usually works in all situations. See https://github.com/zyra/ionic-image-loader#setimagereturntypeimagereturntype-string

In the latest `ionic-image-loader` version, I have made some changes so that the plugin automatically works in the best way possible in any environment. I will do some testing soon to verify that it works in all scenarios (perhaps I should write tests and use cloud testing).

**Comment by jonseg** [comment]
Hi,

It's perfect now changing setImageReturnType to base64.

Great work! Thank you.

**Comment by ihadeed** [comment]
@jonseg could you please tell me if you were using WKWebView or just the regular UIWebView ?

**Comment by jonseg** [comment]
I am using WKWebView.

**Comment by kshaaban-** [comment]
Just sharing what I did to make it work with Ionic's fork of WKWebView, which breaks it for the standard WKWebView plugin one
https://github.com/kshaaban-/ionic-image-loader/commit/d3fa242117f1267f6dba4f0ed9f8640183664f1f

**Comment by ihadeed** [comment]
@kshaaban- thanks. 

I just realized that I made a fix for this awhile ago but I forgot to publish it. I was trying to test it on my apps before publishing and I forgot to push my commits. See https://github.com/zyra/ionic-image-loader/commit/b17e1825c7a862597e3e640647452639d4cbbae0

I will publish it now as 3.2.1

**Comment by ihadeed** [comment]
Just published `ionic-image-loader@3.2.1`. Should fix iOS issues.
